app/models/podcast.rb

```diff
@@ -184,4 +189,26 @@ def sanitize_text
   def feeder_cdn_host
     ENV['FEEDER_CDN_HOST']
   end
+
+  # TODO: temporary delegations, until Publish + our Representers get updated
+  # the tests also seem to have issues with this - and the need for just-in-time
+  # initializing the default feed
+  def method_missing(method, *args, &block)
+    if FEED_GETTERS.include?(method)
+      default_feed.try(:public_send, method, *args, &block)
+    elsif FEED_SETTERS.include?(method)
+      self.default_feed ||= feeds.new(private: false)
```

Comment:
The only thing I could figure out to make the tests and API happy -

1) When any of these delegated field's setter is called, just-in-time create the default feed.
2) If nothing calls these setters on create, _still_ create the default feed up in the `set_defaults` that happens on `before_validate`.